### ISSUE TITLE
Read file permissions

### ISSUE THREAD

**Issue by vitgrid** [davidmwilliams/SecurityExplorer#5]
You would like to see following permissions right?

![image](https://user-images.githubusercontent.com/38117747/96536414-04b50800-1262-11eb-8135-23b73c9d779b.png)

**Comment by davidmwilliams** [comment]
Yes - wow, I appreciate your interest and help.

On a "regular" desktop PC the permissions will look like this. A file server in an enterprise will have a lot more entries, but still the same sort of thing. For example, the "Finance" share might have security groups on it like DOMAIN\Accountants. Using groups for access is recommended because it's easy to add/remove people from groups, and to always know who is in the group.

In practice, every organisation I have gone to has file shares littered with named users (e.g. DOMAIN\Fred Smith) in there and then that person leaves and their replacement doesn't have access and is confused. Or, someone wants to share one single doc from the file share so they get the person added to it and suddenly Fred Smith can see all the payroll information when they're not supposed to.

My idea for this app came after trawling through some massive file shares at one organisation and trying to clean it all up. What I want the app to do is show these permissions and, importantly, show where they vary for files and folders below. Under "Advanced" there is an option to inherit permissions from above, and similarly to push permissions down below. My thinking is we highlight when the security controls have users and/or groups on a file or folder that are in addition to inheritable items from above.

I hope that makes sense!

TL;DR - yes :) And we want to highlight when a file/folder has something different to its parent.
